presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -256,9 +275,12 @@ public Ssl()
         private static final Predicate<Properties> IF_SSL_ENABLED =
                 checkedPredicate(properties -> SSL.getValue(properties).orElse(false));
 
+        private static final Predicate<Properties> IF_SSL_VERIFICATION_ENABLED =
+                checkedPredicate(properties -> SSL_VERIFICATION.getValueOrDefault(properties).isPresent() && !SSL_VERIFICATION.getValueOrDefault(properties).get().equals(SslVerificationMode.NONE));
```

Comment:
```
                checkedPredicate(properties ->  !SSL_VERIFICATION.getValueOrDefault(properties).orElseThrow().equals(SslVerificationMode.NONE));
```

What `SSL_VERIFICATION.getValueOrDefault(properties).isEmpty()` would mean? `NONE` or `FULL`. I think we require a value here, we cannot evaluate this predicate to `false` if value is not set as that would mean `NONE`.

Reply:
`SSL_VERIFICATION` is expected to have a default value.
If `SSL_VERIFICATION.getValueOrDefault(properties).isEmpty()`, should an exception be thrown just like what is done in `PrestoDriverUri`?

```
SslVerificationMode sslVerificationMode = SSL_VERIFICATION.getValueOrDefault(properties).orElseThrow(() ->
                            new SQLException("No default value for connection property " + SSL_VERIFICATION.getKey()));
```

Reply:
> SSL_VERIFICATION is expected to have a default value.

Yes, so `SSL_VERIFICATION.getValueOrDefault(properties).isPresent()` is not needed here.

Reply:
Fixed the code to an exception instead of disallowing `SSL_VERIFICATION`.
`SSL_VERIFICATION.getValueOrDefault(properties).isPresent()` was removed.